ui/src/pages/QuickSetup/QuickSetup.tsx

```diff
@@ -120,30 +121,27 @@ const QuickSetup = withRouter((props: RouteComponentProps<{}>) => {
 
   return (
     <WellKnownPartiesProvider>
-      <div className="quick-setup">
-        <div className="page-controls">
-          <Button className="ghost dark control-button" onClick={() => history.push(paths.login)}>
-            <ArrowLeftIcon color={'white'} />
-            Back
-          </Button>
-          {activeMenuItem !== MenuItems.LOG_IN && (
-            <NavLink to={`${matchUrl}/${MenuItems.LOG_IN}`}>
-              <Button className="button ghost dark control-button">
-                Skip to Log In
-                <ArrowRightIcon color={'white'} />
-              </Button>
-            </NavLink>
-          )}
-        </div>
-
-        <div className="quick-setup-header">
-          <h1 className="logo-header">
-            <OpenMarketplaceLogo size="32" /> Daml Open Marketplace
-          </h1>
-          {activeMenuItem === MenuItems.LOG_IN ? <h2>Log In</h2> : <h2>Market Set-Up</h2>}
-        </div>
-
-        <div className="quick-setup-tile">
+      <Widget
```

Comment:
I pulled out some of the QuickSetup styling into a new component, `Widget`, so I could reuse it for the error page! 

I think this a nice layout for pages "outside" the main app 

![Screen Shot 2021-05-28 at 1 17 50 PM](https://user-images.githubusercontent.com/64022527/120020766-29bbe700-bfb8-11eb-8bb5-4e7e95855b62.png)

Reply:
nice 👍